src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -146,6 +147,13 @@ def _populate_core_stats_and_sim_state(self, area):
                 core_stats_dict['trades'].append(trade.serializable_dict())
         if hasattr(area.current_market, 'market_fee'):
             core_stats_dict['market_fee'] = area.current_market.market_fee
+        if hasattr(area.current_market, 'const_fee_rate') is not None and \
+                area.current_market is not None:
+            core_stats_dict['const_fee_rate'] = area.current_market.const_fee_rate \
+                if area.current_market.const_fee_rate is not None else 0.
+            core_stats_dict['feed_in_tariff'] = FEED_IN_TARIFF
+            core_stats_dict['market_maker_rate'] = \
+                ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE
```

Comment:
I think that it is better to fetch the real current market maker rate from the strategy instead of the constant, reason for that is because there are plans in the future to add market maker profiles, and multiple market makers per simulation.

Reply:
How about using `GlobalConfig.market_maker_rate` ?

Reply:
This will work for now, however keep in mind that this is a dict, therefore you have to access the current market slot value of the market maker rate and not just add the whole dict.